eslint-plugin-expensify/tests/prefer-at.test.js

```diff
@@ -0,0 +1,69 @@
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../prefer-at');
+const message = require('../CONST').MESSAGE.PREFER_AT;
+
+const ruleTester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+    },
+});
+
+ruleTester.run('prefer-at', rule, {
+    valid: [
+        {
+            code: 'example.at(0)',
+        },
+        {
+            code: 'test.at(1)',
+        },
+        {
+            code: 'sample.at(-1)',
+        },
+        {
+            code: 'testing.at(-2)',
+        },
+        {
+            code: 'testing.at(0).at(-1)',
+        },
+        {
+            code: 'test.at((test.length - 1) / 2)',
+        },
```

Comment:
```suggestion
        },
        {
            code: `
                const object = {0: 'v0', 1: 'v1', 2: 'v2'};
                object[0]
                `,
        },
        {
            code: '[0, 1, 2].at(1)',
        },
```

We must assert his rule will be only applied to arrays, not objects